### PR TITLE
Add dusty doctor command

### PR DESCRIPTION
**Pull request by thieman**
@paetling Closes #654.

### CODE REVIEW

File: dusty/commands/doctor.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    delete_docker_vm_host_only_interface()
+    ensure_docker_vm_is_started()
+    regenerate_docker_vm_certificates()
+    # TODO: Remove this step once initialize_docker_vm is smarter
+    # The problem is we couldn't do our standard initialization
+    # steps (making our symlinks, installing rsync, etc) since
+    # the network was having problems. We need to shut the VM
+    # down so the next dusty up attempts to do these. Instead
+    # of basing init off VM state, we should write a file somewhere
+    # and base the operation off of that.
```
Comment (paetling) [inline dusty/commands/doctor.py]: you want to remove regenerate or the shut_down_docker_vm?

Comment (thieman) [inline dusty/commands/doctor.py]: shut down

File: dusty/systems/virtualbox/__init__.py [hunk 2]
```diff
@@ -140,6 +140,22 @@ def _apply_nic_fix():
...
     check_call_demoted(['VBoxManage', 'modifyvm', constants.VM_MACHINE_NAME, '--nictype1', constants.VM_NIC_TYPE])
 
+def delete_docker_vm_host_only_interface():
+    """Attempt to delete the host-only interface attached
+    to the current Dusty VM. VM should be stopped
+    before calling this."""
+    adapter_name = get_vm_hostonly_adapter()
+    log_to_client('Deleting host-only interface {}'.format(adapter_name))
+    check_call_demoted(['VBoxManage', 'hostonlyif', 'remove', adapter_name])
+
```
Comment (paetling) [inline dusty/systems/virtualbox/__init__.py]: there is only 1 host only interface?

Comment (thieman) [inline dusty/systems/virtualbox/__init__.py]: yeah the other one is the forwarding interface that lets you talk to the internet

**Comment by paetling** [comment]
:ghost: :cow:

**Pull request merged**